actionpack/lib/action_controller/renderer.rb

```diff
@@ -60,7 +60,7 @@ def with_defaults(defaults)
     end
 
     # Accepts a custom Rack environment to render templates in.
-    # It will be merged with ActionController::Renderer.defaults
+    # It will be merged with +ActionController::Renderer::DEFAULTS+
```

Comment:
`defaults` is an attr_reader so `ActionController::Renderer.defaults` will work.